src/api/comments.ts

```diff
@@ -0,0 +1,22 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getCommentsById = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const deleteCommentById = (commentId: number) => {
+  return client.delete(`/comments/${commentId}`);
+};
+
+// export const createComment = (data: Comment) => {
+//   return client.post<Comment>('/comments', data);
+// };
+
+export const createComment = ({
+  name, email, body, postId,
+}: Omit<Comment, 'id'>) => {
+  return client.post<Comment>('/comments', {
+    name, email, body, postId,
+  });
+};
```

Comment:
```suggestion
export const createComment = (comment: Omit<Comment, 'id'>) => {
  return client.post<Comment>('/comments', comment);
};
```
Don't leave comments, and you don't need destructing in this case

Reply:
If I use this shortcut, the comment will not be created on the server and I get an error

Reply:
IMO - You are not sure what data would actually be passed to this function - so explicitly sending only the required data is OK